packages/ftp/lib/FtpClient.js

```diff
@@ -56,7 +56,7 @@ class FtpClient {
       },
     })
 
-    const finished = require('readable-stream').finished
+    const { finished } = await import('readable-stream')
```

Comment:
Can this be imported directly at the top of the file, with the other things from `readable-stream`?

Reply:
Yea. Actually I don't know why this was loaded dynamically in the first place 🤔

Reply:
Ok :) You marked it as resolved ; this mean you will push a change or keep it as it?

Reply:
Will change